force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -206,10 +273,10 @@ public with sharing class VariablesDataTypesOperators {
         // Add parentheses in the below expression to change the result.
         // The result of the expression as it is right now is 43. 
         // You should add parentheses so that the result of the expression becomes 8.
-        Integer answer = 48 - 15 + 5 * 2;
-        return null; // Replace null with the variable you used to store the result
+        Integer answer = 48 - ((15 + 5) * 2);
```

Comment:
This is correct but because of the order of operations, we don't need that second set of parantheses.
```suggestion
        Integer answer = 48 - (15 + 5) * 2;
```